pylintrc

```diff
@@ -94,13 +94,14 @@ confidence=
 # no Warning level messages displayed, use"--disable=all --enable=classes
 # --disable=W"
 disable=
-    no-init,
     fixme,
     locally-disabled,
+    no-init,
     too-few-public-methods,
     too-many-ancestors,
-    unused-argument,
-    protected-access
+    too-many-instance-attributes,
```

Comment:
@antoviaque I've never found this particular warning useful, so I wasn't shy disabling it globally.